hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -535,6 +535,13 @@
       "ozone.scm.ca.list.retry.interval";
   public static final long OZONE_SCM_CA_LIST_RETRY_INTERVAL_DEFAULT = 10;
 
+
+  public static final String OZONE_SCM_EVENT_PREFIX = "ozone.scm.event.";
+
+  public static final String OZONE_SCM_EVENT_CONTAINER_REPORT_THREAD_POOL_SIZE =
+      OZONE_SCM_EVENT_PREFIX + "ContainerReport.thread.pool.size";
+  public static final int OZONE_SCM_EVENT_THREAD_POOL_SIZE_DEFAULT = 10;
+
```

Comment:
OZONE_SCM_EVENT_THREAD_POOL_SIZE_DEFAULT -> OZONE_SCM_EVENT_CONTAINER_REPORT_THREAD_POOL_SIZE_DEFAULT??

Reply:
As the class SingleThreadExecutor is common for all Events, I have named the default variable in a generic way, but the config name construction is done from the event name. As we have more events which need different defaults, this can be revisited if needed.